docs/converters/tabular_converter.md

```diff
@@ -53,13 +52,12 @@ substitutions:
 ```
 
 ## Grid
-For each `table`, the target PGM `component` is listed (e.g. Nodes: node, Cables: line).
-The for each PGM `column` the source column is supplied (e.g. u_rated: Unom, from_status: From.SwitchStatus).
+For each `table`, the target PGM `component` is listed (e.g., Nodes: node, Cables: line).
```

Comment:
don't think we need a comma here (and line below)

Reply:
Haha, this also used to confuse me.  For `e.g.`, a comma is always required after it due to its meaning and its application in English sentences. The comma after `i.e.` on the other hand, is optional depending on the context.

Reply:
english is hard https://www.youtube.com/watch?v=tfRSvTSY0d4